roles/ceph-osd/tasks/activate_osds.yml

```diff
@@ -41,6 +41,18 @@
 - include: osd_fragment.yml
   when: crush_location
 
+- name: check if selinux is enabled
+  command: getenforce
+  register: sestatus
+  changed_when: false
```

Comment:
There should also be an ansible_selinux_config_mode fact available to avoid the need for calling out to getenforce.